components/Header/Navigation.jsx

```diff
@@ -36,7 +43,14 @@ function classNames(...classes) {
 }
 
 export default function Navigation() {
+  const [showCalendar, setShowCalendar] = useState(false);
+
+  const handleCalendarClick = () => {
```

Comment:
@daynamariejpembie sorry if I wasn't explicit on this. 

But basically we only need here is a new Page with the calendar, where the users could go back and forward from it. 

Lets create a new Page and call it Calendar, add the Iframe there, then we should be good to go there.

Reply:
@jonathans199 I'm taking a look at this to help @daynamariejpembie. To be clear, what you're talking about doing is making a `calendar.js` page under the pages directory right? And that would just contain the iframe. Then there needs to be ternary (or some other way of bringing the calendar to view) inside `components/Header/Navigation.jsx` that disables the mobile or medium and up sized buttons and then renders the calendar? The second part is tripping me up in particular, and that may not be how it needs to rendered. Please elaborate so we can implement this, thanks!